package.json

```diff
@@ -51,6 +51,7 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.19.1",
+    "@testing-library/react": "^12.1.2",
```

Comment:
We plan on adding this later but since your tests dont use it now can you remove it from this PR?

Reply:
done